pymeasure/instruments/signalrecovery/dsp7265.py

```diff
@@ -199,19 +219,12 @@ def __init__(self, resourceName, **kwargs):
             resourceName,
```

Comment:
would you mind moving the __init__ to the top of the class definition? I always find it frustrating to hunt for it.

Reply:
Sure